### DISCUSSION TITLE
Filter by relation data (contains)

### DISCUSSION THREAD

**Discussion by muhammed-kilinc** [pocketbase/pocketbase#2157] [Q&A] [answered]
Hello 
How can I create a filter like this:

- Channel
- - Users
- - - User1
- - - User2

Select all channels where users includes user1 
Users is a multiple relation

**Comment by ganigeorgiev** [comment, marked as answer]
If `channel` is a collection with multi-`relation` field named `users`, then the following filter should work:

```js
users.id ?= "YOUR_USER1ID"

// OR if you want to match with the authenticated user
@request.auth.id != "" && users.id ?= @request.auth.id
```

The operator `?=` translates to "any/at least one of" the id should be equal to the right operand.

If the operator was just `=` then it will require **all** `relation` items to match the right operand. This is useful usually if you want to check non-id fields. For example the below rule will require **every** channel users to have a role "member":
```js
users.role = 'member'
```
